airsonic-main/src/main/webapp/WEB-INF/jsp/podcastChannel.jsp

```diff
@@ -59,7 +162,13 @@
             location.href = "podcastChannel.view?id=${model.channel.id}";
         }
 
-        function getSelectedEpisodes() {
+        function selectAll(checked) {
+            for (var i = 0; i < ${fn:length(model.episodes)}; i++) {
+                $("#episode" + i).prop("checked", checked);
+            }
+        }
+
+        function getSelectedEpisodesId() {
```

Comment:
i think maybe just keep the name to reduce noise

Reply:
As I had the name getSelectedEpisodesMediaId to get the media Id, I tought it would make sense to have the Id there as well (as it actually returns ids).
I'm going to revert that.